src/Symfony/Component/Runtime/SymfonyRuntime.php

```diff
@@ -143,12 +147,23 @@ public function __construct(array $options = [])
 
         $options['error_handler'] ??= SymfonyErrorHandler::class;
 
+        $workerLoopMax = $options['worker_loop_max'] ?? $_SERVER['FRANKENPHP_LOOP_MAX'] ?? $_ENV['FRANKENPHP_LOOP_MAX'] ?? null;
+        if (null !== $workerLoopMax && (!\is_numeric($workerLoopMax) || $workerLoopMax <= 0)) {
```

Comment:
`filter_var`?

Reply:
If we do this (for example `filter_var($options['worker_loop_max'] ?? $_SERVER['FRANKENPHP_LOOP_MAX'] ?? $_ENV['FRANKENPHP_LOOP_MAX'] ?? null, FILTER_VALIDATE_INT, FILTER_NULL_ON_FAILURE);`), we cannot detect if the option is a string like "foo", and it'll fallback on the default value. It works, but there will be no explicit error message to warn the user. Maybe I'm missing an argument to provide to filter_var?

Reply:
The code has `-1 === $this->loopMax` so this check is not correct.
BTW, It'd suggest turning this into `0 >= $this->loopMax`